### PR TITLE
Merge my work from over the weekend

### PR DESCRIPTION
**Pull request by tkpanke**
Merge my work from over the weekend with stacktrace

### CODE REVIEW

**Comment by OneLeggedPigeon** [comment]
@tkpanke did you check with @RevatureRobert/styling for you changes to style.css?

File: src/app/stacktrace/components/new-stacktrace/new-stacktrace.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="new-stacktrace-wrapper" fxLayout="row" fxLayoutGap="20px" fxLayoutAlign="center center">
+  <mat-card class="box">
```
Comment (pickettmap) [inline src/app/stacktrace/components/new-stacktrace/new-stacktrace.component.html:2]: We aren't using Angular material. Change this to bootstrap card

**Review by OneLeggedPigeon — APPROVED** [review]
Everything looks good to me, but let's wait till styling reviews style.css before merging.

File: src/app/stacktrace/stacktrace.module.ts [hunk 2]
```diff
@@ -2,13 +2,23 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { StacktraceHomeComponent } from './components/stacktrace-home/stacktrace-home.component';
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+import { NewStacktraceComponent } from './components/new-stacktrace/new-stacktrace.component';
+import { MatCardModule } from '@angular/material/card';
```
Comment (pickettmap) [inline src/app/stacktrace/stacktrace.module.ts:6]: Remove this

File: src/styles.css [hunk 3]
```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+
+html, body { height: 100%; }
```
Comment (pickettmap) [inline src/styles.css:3]: Don't change the styles.css. For now, keep them on a component basis within your [component].css. If it shows that we do need to incorporate this as a global style, we can alter it then

File: src/index.html [hunk 4]
```diff
@@ -6,8 +6,11 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (pickettmap) [inline src/index.html:11]: Remove this

**Review by pickettmap — CHANGES_REQUESTED** [review]
Switch from angular materials to bootstrap. Remove the changes to styles.css and apply those rules on a component basis for now.

**Pull request merged**